src/css.js

```diff
@@ -413,19 +413,32 @@ const styles = `
     cursor: pointer;
   }
 
-  .scratch-renderer.scratch-theme .blocklyFlyoutLabelText {
+  .scratch-renderer.default-theme .blocklyFlyoutLabelText,
+  .scratch-renderer.high-contrast-theme .blocklyFlyoutLabelText {
     font-family: "Helvetica Neue", Helvetica, sans-serif;
     font-size: 14pt;
     fill: #575E75;
     font-weight: bold;
   }
 
-  .scratch-renderer.scratch-theme .blocklyText,
-  .scratch-renderer.scratch-theme .blocklyHtmlInput {
+  .scratch-renderer.default-theme .blocklyText,
+  .scratch-renderer.default-theme .blocklyHtmlInput,
+  .scratch-renderer.high-contrast-theme .blocklyText,
+  .scratch-renderer.high-contrast-theme .blocklyHtmlInput  {
     font-weight: 500;
   }
 
-  .scratch-renderer.scratch-theme .blocklyFlyoutButton .blocklyText {
+  .scratch-renderer.high-contrast-theme .blocklyText,
+  .scratch-renderer.high-contrast-theme .blocklyEditableField .blocklyDropdownText {
+    fill: #000 !important;
```

Comment:
Why does this have to use `!important`?

Reply:
Because of this: https://github.com/google/blockly/blob/develop/core/renderers/zelos/constants.ts#L842-L845 😒